cms/src/components/content/EditContentParent.spec.ts

```diff
@@ -27,10 +27,28 @@ describe("EditContentParent.vue", () => {
         });
 
         // // Check if the LToggle component is rendered
-        const toggle = wrapper.findComponent({ name: "LToggle" });
+        const toggle = wrapper.find('[data-testId="pinned-LToggle"]');
         expect(toggle.exists()).toBe(true);
 
-        expect(toggle.props("modelValue")).toBe(true);
+        const props = toggle.getCurrentComponent()?.props;
+        expect(props!["modelValue"]).toBe(true);
+    });
+
+    it("test the link date toggle", async () => {
```

Comment:
I see the other tests in this file have similar names, but I think they can be written a bit more clearly. Generally I would recommend writing it in a sentence form (it already starts with "it", so write whatever comes after that: "it does X when I click Y" for example). And when this test fails, a clear description of what this test does helps to view at a glance what might be wrong. My suggestion for this specific case would be something like "it renders the link dates toggle with the correct initial state". 

Do you agree that makes it clearer? In that case I suggest to also rewrite the other test names in this file.

On a separate note, we could perhaps add a test or assertion in this same test that we can toggle it, and that it updates the right value on the parent.